app/common/services/maps.js

```diff
@@ -103,6 +103,76 @@ function Maps(ConfigEndpoint, L, _, CONST) {
             if (config.default_view.baselayer === 'MapQuest') {
                 config.default_view.baselayer = 'streets';
             }
+            return deferred.promise;
+        },
+        reloadMapConfig: function () {
```

Comment:
It looks like this has brought in some old code? this function isn't used or needed